unstructured/ingest/interfaces.py

```diff
@@ -38,6 +40,13 @@
 ]
 
 
+EMBEDDING_PROVIDER_TO_CLASS_MAP = {
```

Comment:
We might want to add this map to the `unstructured.embed` directory so that code outside of ingest could leverage it as well.

Reply:
Address with [move mapping object to embed module](https://github.com/Unstructured-IO/unstructured/pull/1918/commits/921dd56b83c8c35d4b05ecd988bff120d02bafa9)

Why would we need this outside of ingest, won't python users import the classes directly? Do we assume there'll be other tools which'll need a mapping?